riscv_isa.cpp

```diff
@@ -17,7 +17,7 @@
 #include "math.h"
 
 // Uncomment for debug Information
-//#define DEBUG_MODEL
+  #define DEBUG_MODEL
```

Comment:
It's better to leave DEBUG_MODEL disabled in all your commits. If the user wants a super verbose output, it's just a matter of changing this line.